### PR TITLE
Shorted text block under Analysis page title

### PR DESCRIPTION
**Pull request by brifemu**
In this PR, I have shortened the text block under the title on Analysis page. The second part of the text block had been inserted on a modal message, looking like: 

**English**

image
image

**German**

image
image

### CODE REVIEW

File: src/data/analyse_en.json [hunk 1]
```diff
@@ -4,10 +4,9 @@
 	"topic": "Analysis",
     "title": "Corona-Warn-App (CWA): Key performance indicators",
     "update": "Last update of the data:",
-    "updateNotice": "The CWA dashboard data is usually updated daily at 10:30 AM (CET) for the previous day.",
     "updateNoticeIcon": {
         "title": "Update of the data",
-        "content": "Downloads data is updated only on working days. On Mondays, the data for Friday, Saturday and Sunday are updated."
+        "content": "The CWA dashboard data is usually updated daily at 10:30 AM (CET) for the previous day. Downloads data is updated only on working days. On Mondays, the data for Friday, Saturday and Sunday are updated."
```
Comment (MikeMcC399) [inline src/data/analyse_en.json:9]: I suggest instead of

"Downloads data ..."

change to

"Data regarding downloads ..."

This sounds better in English.

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
I made a suggestion for text change in English.

File: src/data/analyse_en.json (hunk 1 above)
Comment (brifemu) [reply src/data/analyse_en.json]: Done, thank you!

**Comment by dsarkar** [comment]
@MikeMcC399 Thanks for reviewing.

**Comment by MikeMcC399** [comment]
@dsarkar 
> Thanks for reviewing.

As always, you are very welcome!

**Pull request merged**